modules/data.land/tests/testthat/test-match_species_id.R

```diff
@@ -0,0 +1,41 @@
+test_that("Species matching works", {
```

Comment:
Not actually a new file, just renamed from `test.match_species_id.R`. Apparently the rearragements I did in other commits before the move are enough to make gitHub treat this and a delete-and-add rather than a rename.